soul-admin/pom.xml

```diff
@@ -28,7 +28,7 @@
     <artifactId>soul-admin</artifactId>
 
     <properties>
-        <nacos-client.version>1.2.0</nacos-client.version>
+        <nacos-client.version>2.0.0</nacos-client.version>
```

Comment:
maybe move to root pom.xml

Reply:
it only change version of  nacos-client to provide performance of nacos.

commit c5bbe3c fix error when nacos username or password is null in config file.